contracts/starknet/authenticator/authenticator.cairo

```diff
@@ -0,0 +1,15 @@
+%lang starknet
+from starkware.starknet.common.syscalls import call_contract
+
+# Forwards `data` to `target` without verifying anything.
+# TODO: use ADDRESS instead of felt
```

Comment:
ive got an address type implemented so we can change soon

Reply:
What the use of Address type here? Isn't Address type used only for L1 addresses?

Reply:
We could use L1Address and StarknetAddress :) I'm in favor of replacing as many `felt` as possible, because otherwise we end up with a code that is just full of `felt`s :p

Reply:
Currently since there's no way to actually wrap a type (other than creating a struct), I'm not sure it makes sense to actually use `StarknetAddress`. Therefore I'm removing this comment :)